mindsdb/integrations/handlers/youtube_handler/youtube_tables.py

```diff
@@ -96,14 +99,68 @@ def select(self, query: ast.Select) -> pd.DataFrame:
 
         return youtube_comments_df
 
+    def update(self, query: ast.Update) -> pd.DataFrame:
+        """
+        Inserts a new comment in the youtube comments table.
+
+        Args:
+            updated message
+            video_id: The video's unique identifier.
+            comment_id: The comment's unique identifier or ID.
+
+        Returns:
+            Response: Response object containing the results.
+        """
+        update_statement_parser = UPDATEQueryParser(query, self.get_columns())
+        (
+            selected_columns,
+            where_conditions,
+        ) = update_statement_parser.parse_query()
+        params = {}
+        for op, arg1, arg2 in where_conditions:
+            if arg1 == "video_id":
+                if op == "=":
+                    params[arg1] = arg2
+                else:
+                    raise NotImplementedError(f"Invalid operator, {op}. Only the '=' operator is currently supported")
+            if arg1 == "comment_id":
+                if op == "=":
+                    params[arg1] = arg2
+                else:
+                    raise NotImplementedError(f"Invalid operator, {op}. Only the '=' operator is currently supported")
+
+        if "video_id" not in params or "comment_id" not in params:
+            raise Exception(
+                "To update a comment in Youtube, you need to provide the 'video_id' and comment_id parameter."
+            )
+        # Update comment in YouTube video
+        try:
+            body = {"id": params["comment_id"], "snippet": {"textOriginal": selected_columns["comment"]}}
+            update_response = (
+                self.handler.connection.comments()
+                .update(
+                    part="snippet",
+                    body=body,
+                )
+                .execute()
+            )
+            return f"Video comment {params['comment_id']} updated"
+        except HTTPError as e:
+            if e[0]["reason"] == "forbidden":
+                logger.error(
+                    "Due to ownership restrictions, you are unable to update this comment on the YouTube server."
+                )
+            else:
+                logger.error(f"Unexpected error encountered while updating comments {e}")
+
     def get_columns(self) -> List[str]:
         """Gets all columns to be returned in pandas DataFrame responses
         Returns
         -------
         List[str]
             List of columns
         """
-        return ["user_id", "display_name", "comment", "replies"]
+        return ["comment_id", "user_id", "display_name", "comment"]
```

Comment:
It looks like these columns were placed as a result of the initial implementation of this integration. Was this incorrect? Do SELECT queries still run successfully?